# Get current robot angle

---
float64 angle
bool success
string msg
